### ISSUE TITLE
Show star rating / review summary in review listing

### ISSUE THREAD

**Issue by kumar303** [mozilla/addons-frontend#3919]
The Add-on review listing should show a summary of star ratings and reviews to help users better assess the quality of the add-on before installing it.

Please check the mocks before implementing but currently it looks like:

<img width="508" alt="screenshot 2017-11-15 11 56 37" src="https://user-images.githubusercontent.com/55398/32852024-5a497ef6-c9fc-11e7-83cc-84980ed97b43.png">

This will look similar to the summary shown in detail pages: https://github.com/mozilla/addons-frontend/issues/3918

**Comment by pwalm** [comment]
Design updated and referenced in this issue: https://github.com/mozilla/addons-frontend/issues/5462

**Comment by kumar303** [comment]
@pwalm : The current listing page has clickable stars for the user to select their rating ([example](https://addons.mozilla.org/en-US/firefox/addon/emoji-cheatsheet/reviews/)). I'm not sure that we can still use a star summary for that and if we can, I think it needs some changes to make it obvious that the stars are clickable.

**Comment by kumar303** [comment]
We discussed this on IRC. The old listing page let you create/edit a rating or review. @pwalm says the new listing page *should not*. It should be the read-only view from the specs.

**Comment by pwalm** [comment]
Correct! Read only everywhere except the rating block on the detail page

**Comment by ioanarusiczki** [comment]
- the rating block is only editable from the detail page once clicked https://addons-dev.allizom.org/en-US/firefox/addon/adblock-edge/reviews/ the page has just read only properties.
- stars are increased compared to the old layout.
- the user must click the Read All link to see the review, the same when it is edited.

Verified fixed on AMO dev with FF61(Win10&Android 8.0)
